packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1254,6 +1240,9 @@ class _LineItemState extends State<LineItem>
     );
   }
 
+  Color _focusedColor(Color color, {required bool isDarkTheme}) =>
+      isDarkTheme ? color.brighten() : color.darken();
```

Comment:
can we use `colorScheme.selectedRowBackgroundColor` instead of manually brightening or darkening the canvasColor?

Reply:
Done!